docs/language.rst

```diff
@@ -125,7 +125,17 @@ Literal values can be assigned to variables
 Primitive types
 ==============================
 
-The basic primitive types are ``string``, ``number``, ``int`` or ``bool``.
+The basic primitive types are ``string``, ``number``, ``int`` or ``bool``. These basic types also support type casts:
+
+.. code-block:: inmanta
+
+    assert = true
+    assert = int("1") == 1
+    assert = number("1.2") == 1.2
+    assert = number(true) == 1
+    assert = bool(1.2) == true
+    assert = bool(0) == false
```

Comment:
How about bool conversion of a string? I would like `bool("false") == false` but if it works like python this won't be the case

Reply:
(I just tested it and indeed it does work like in python)

Reply:
It indeed just delegates casting to the corresponding Python types.